autopush/endpoint.py

```diff
@@ -43,11 +44,26 @@ def _load_params(self):
         self.data = data
 
     @cyclone.web.asynchronous
+    def options(self, token):
+        self._addCors()
+        self.set_status(200)
+        self.write("")
+        self.finish()
```

Comment:
These don't need to be asynchronous since when your function returns, its actually done. So the decorator can be dropped, along with the calls to write and finish, and set_status (200 is default, finish isn't needed without asynchronous, and I don't believe you need to write anything).